growth/too/models.py

```diff
@@ -47,7 +47,8 @@ def get_ztf_quadrants():
 def create_all():
     db.create_all(bind=None)
 
-    telescopes = ["ZTF", "Gattini", "DECam", "KPED", "GROWTH-India"]
+    # telescopes = ["ZTF", "Gattini", "DECam", "KPED", "GROWTH-India"]
```

Comment:
Uh oh. We don't want to delete the other telescopes, do we?

Reply:
OK, this should be fixed now.